src/python/dxpy/system_requirements.py

```diff
@@ -0,0 +1,182 @@
+
+from collections import defaultdict
+import copy
+
+from .utils import merge
+from .compat import basestring
+from .exceptions import (err_exit, DXError, DXCLIError)
+
+'''
+System Requirements
++++++++++++++++++++
+
+A module containing utility methods useful for packing and unpacking
+system requirements.
+'''
+
+class SystemRequirementsDict(object):
+    def __init__(self, instance_type=None, cluster_spec=None):
+        self.instance_type = instance_type
```

Comment:
Might be good to document what `instance_type` and `cluster_spec` look like using small examples